Makefile

```diff
@@ -8,4 +8,5 @@ ifneq (,$(shell grep "path *= *$(LIBDIR)" .gitmodules 2>/dev/null))
 else
 	git clone -q --depth 10 $(CLONE_ARGS) \
 	    -b main https://github.com/martinthomson/i-d-template $(LIBDIR)
+	(cd $(LIBDIR) && git checkout 3e5cfbafc0037fee41e7fe4e6567b14b3c1be199) # Temporarily pin to a specific commit right before it upgraded mmark from 2.2.10 to 2.2.40 as the latter is producing XML from the current markdown that xml2rfc rejects
```

Comment:
Well, that didn't work. Apparently I don't understand the github workflow/action build process well enough.

Reply:
The dependencies (like mmark) seem to be installed in the docker container of the github action -> You would need to modify the action version to switch to an older mmark version from what I understand?

Reply:
Maybe? I'm honestly not sure how to do that or even which action or how/if its versioned. 

https://github.com/martinthomson/i-d-template/pull/412/files is the change to action/Dockerfile in the martinthomson/i-d-template FWIW. 

I was just trying for quick workaround here though. And chasing down the main issue later. But maybe a quick workaround isn't doable.

Reply:
I clearly have no idea what I'm doing ... :(